TestSuite/repo.py

```diff
@@ -208,7 +213,7 @@ def setup_one_pack(self, name, marketplace='xsoar') -> Pack:
         pack.create_job(is_feed=False, name=name)
         pack.create_job(is_feed=True, name=f'{name}_all_feeds')
 
-        if marketplace != 'xsoar':
+        if marketplaces == ['marketplacev2']:
```

Comment:
Should be like `if 'marketplacev2' in marketplaces`

Reply:
done